### PR TITLE
Update import and fix indent

### PR DESCRIPTION
**Pull request by tao12345666333**

### CODE REVIEW

File: create_aio_app/utils/generator.py [hunk 1]
```diff
@@ -1,18 +1,12 @@
-from distutils.dir_util import copy_tree
-import shutil
 import os
+import shutil
+from distutils.dir_util import copy_tree
+
+from jinja2 import (BaseLoader, Environment)
```
Comment (Arfey) [inline create_aio_app/utils/generator.py]: I prefer the old style for `import`. Can u reformat it?

```python
from jinja2 import (
    BaseLoader,
    Environment,
)
```

This style helpful if u watch git diff

**Comment by tao12345666333** [comment]
@Arfey  Addressed. Thanks.

File: create_aio_app/utils/generator.py [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
+    Environment
 )
 
-from create_aio_app.constants import (
-    TEMPLATE_DIR,
-    TEMPLATE_NAME,
-    DATABASE_TEMPLATE_DIRS,
-)
-from create_aio_app.constants import RENDER_ACCESS_FORMATS
+from create_aio_app.constants import (DATABASE_TEMPLATE_DIRS,
```
Comment (Arfey) [inline create_aio_app/utils/generator.py]: `jinja2` i used as example 😅. Can u corrected all imports in file?

Comment (tao12345666333) [reply create_aio_app/utils/generator.py]: sorry, it's my mistake.

**Comment by tao12345666333** [comment]
@Arfey  Thanks for your reply. Addressed.

**Comment by Arfey** [comment]
thank you!

**Pull request merged**